dotnet-logging/AuditLogging/AuditLogApp/Models/AuditLog.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace AuditLogApp.Models;
+
+public class AuditLog
+{
+    public int Id { get; set; }
+    public string UserEmail => "John.Doe@gmail.com";
+    public required string EntityName { get; set; }
+    public required string Action { get; set; }
+    public DateTime Timestamp { get; set; }
```

Comment:
Shouldn't the `Timestamp` also be `required`? I believe we should have timestamp for all `AuditLog` records.